x-pack/plugins/siem/public/components/page/overview/overview_network/index.tsx

```diff
@@ -22,9 +28,30 @@ export interface OwnProps {
 export const OverviewNetwork = pure<OwnProps>(({ endDate, poll, startDate }) => (
   <EuiFlexItem>
     <EuiPanel>
-      <EuiTitle>
-        <h2>{i18n.NETWORK_HEADING}</h2>
-      </EuiTitle>
+      <EuiFlexGroup alignItems="center">
+        <EuiFlexItem>
+          <EuiTitle>
+            <h2>
+              <FormattedMessage
+                id="xpack.siem.overview.networkTitle"
+                defaultMessage="Network Ingest Indices"
+              />
+            </h2>
+          </EuiTitle>
+        </EuiFlexItem>
+
+        <EuiFlexItem grow={false}>
+          <EuiButton href={`#/link-to/network/`}>
```

Comment:
nit: You can do: <EuiButton href="#/link-to/network/">

Reply:
On it!